include/valkey/sds.h

```diff
@@ -35,7 +35,8 @@
 
 #define SDS_MAX_PREALLOC (1024*1024)
 #ifdef _MSC_VER
-typedef long long ssize_t;
+#include <basetsd.h>
+typedef SSIZE_T ssize_t;
 #define SSIZE_MAX (LLONG_MAX >> 1)
```

Comment:
Should `SSIZE_MAX` also be changed in some similar way?
LLONG_MAX seems to big on your 32bit..or maybe it will "just work" ?

Reply:
Oh yeah thanks, good catch. I'll fix it.

Reply:
Fixed